nsxt/resource_nsxt_policy_ipsec_vpn_service_test.go

```diff
@@ -103,6 +103,73 @@ func TestAccResourceNsxtPolicyIPSecVpnService_basic(t *testing.T) {
 	})
 }
 
+func TestAccResourceNsxtPolicyIPSecVpnService_withGateway(t *testing.T) {
+	testResourceName := "nsxt_policy_ipsec_vpn_service.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t); testAccOnlyLocalManager(t) },
```

Comment:
I think version check is needed here, since GW api was not exposed for early versions (3.2.0?)